docs/advanced-search.md

```diff
@@ -0,0 +1,339 @@
+# Advanced Search
+
+```eval_rst
+.. contents:: :local:
+```
+
+Kitsune supports an advanced search syntax in all its search boxes.
+
+A number of search tokens,
```

Comment:
Maybe add a couple of lines defining what token is since it's referenced in the rest of the text?